src/material.h

```diff
@@ -75,4 +77,13 @@ class material_type
         const mat_burn_data &burn_data( size_t intensity ) const;
 };
 
+namespace materials
```

Comment:
Can you site examples in the codebase or that we might realistically implement soon otherwise this adds needless indirection: YAGNI.